source/common/upstream/upstream_impl.cc

```diff
@@ -317,11 +327,12 @@ HostSetImpl::partitionHosts(HostVectorConstSharedPtr hosts,
                            std::move(degraded_hosts), std::move(degraded_hosts_per_locality));
 }
 
-double HostSetImpl::effectiveLocalityWeight(uint32_t index) const {
+double HostSetImpl::effectiveLocalityWeight(uint32_t index,
+                                            const HostsPerLocality& eligible_hosts) const {
```

Comment:
s/eligible_hosts/eligible_hosts_per_locality/

Should be obvious by the function name but we have so many things floating around I'd rather be explicit